content/en/author/tianshu-li.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Tianshu Li"
+image: ""
+email: "tianshu@rotational.io"
+social:
+  - icon : "ti-linkedin" # themify icon pack : https://themify.me/themify-icons
+    link : "https://www.linkedin.com/in/tianshu-li-5b5563b0/"
+  - icon : "ti-github" # themify icon pack : https://themify.me/themify-icons
+    link : "https://github.com/tl6kk"
+---
+
+
+Data scientist and NLP specialist with a PhD in Systems Engineering and an enduring curiosity for learning, open source, and new culinary adventures.
```

Comment:
Feel free to edit 😉  just wanted to make sure we gave a hint into your personality here!